src/pjs/pjs.proto

```diff
@@ -192,11 +192,30 @@ message ListJobResponse {
   JobInfoDetails details = 3;
 }
 
+enum WalkAlgorithm {
+  LEVEL_ORDER = 0; // similar to BFS, but specifically for trees. Traverse nodes in height order.
```

Comment:
Something that happens with protos is when a client has a 4th algorithm added and the server doesn't have that algorithm (compiled against older protos), it will be mapped to the first entry here.  So LEVEL_ORDER will be silently selected.

The way to avoid this is to call the first entry UNKNOWN and then never use it.  If you get a request where the algorithm is UNKNOWN, you'll know that something is out of sync.

Reply:
Now I know why we always use UNKOWN in enums.